pkg/operator/ceph/file/subvolumegroup/controller.go

```diff
@@ -219,20 +219,27 @@ func (r *ReconcileCephFilesystemSubVolumeGroup) reconcile(request reconcile.Requ
 		return reconcile.Result{}, nil
 	}
 
+	// Build the NamespacedName to fetch the Filesystem and make sure it exists, if not we cannot
+	// create the subvolume group
+	cephFilesystem := &cephv1.CephFilesystem{}
+	cephFilesystemNamespacedName := types.NamespacedName{Name: cephFilesystemSubVolumeGroup.Spec.FilesystemName, Namespace: request.Namespace}
+
 	if cephCluster.Spec.External.Enable {
-		logger.Debug("external subvolume group creation is not supported, create it manually, the controller will assume it's there")
+		logger.Debug("skip creating external subvolume in external mode, create it manually, the controller will assume it's there")
```

Comment:
Isn't this being created on line 235 below? Or else I'm just confused about the message.

Reply:
same as previous comment